syft/generic/frameworks/hook/hook.py

```diff
@@ -371,24 +371,63 @@ def overloaded_native_method(self, *args, **kwargs):
             """
 
             if not hasattr(self, "child"):  # means that it's not a wrapper
+
+                # if self is a natural tensor but the first argument isn't,
+                # wrap self with the appropriate type and re-run
+                if len(args) > 0 and hasattr(args[0], "child"):
+
+                    # if we allow this for PointerTensors it opens the potential
+                    # that we could accidentally serialize and send a tensor in the
+                    # arguments
+                    if not isinstance(args[0].child, PointerTensor):
+                        self = type(args[0].child)().on(self, wrap=True)
+                        args = [args[0]]
+                        return overloaded_native_method(self, *args, **kwargs)
```

Comment:
Rename
arg = args[0]

Simplify
```
args = [args[0]]
return overloaded_native_method(self, *args, **kwargs)
```
`return overloaded_native_method(self, arg, **kwargs)`
should be the same

Reply:
done